expense/src/server/index.ts

```diff
@@ -1,21 +1,19 @@
-'use strict';
+import express from 'express';
 
-const express = require('express');
-
-const expenseState = Object.freeze({
-  created: 'CREATED',
-  approved: 'APPROVED',
-  rejected: 'REJECTED',
-  completed: 'COMPLETED'
-})
+enum expenseState {
+  created = 'CREATED',
+  approved = 'APPROVED',
+  rejected = 'REJECTED',
+  completed = 'COMPLETED'
+}
 
 run().catch(err => console.log(err));
```

Comment:
You should exit the process with non 0 code in this case.
Also, best practice is to log to stderr.